DFC.Digital/DFC.Digital.Data/Model/CompositeUI/SocSkillMatrixItem.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DFC.Digital.Data.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace DFC.Digital.Data.Model
+{
+    public class SocSkillMatrixItem : IDigitalDataModel
+    {
+        private const string UrlNameRegexPattern = @"[^\w\-\!\$\'\(\)\=\@\d_]+";
```

Comment:
Why do we need this?

Reply:
Needed to create URL. so left it as its private one

Reply:
We should not compute here, should only retrieve source value, hence can be removed?

Reply:
Removed this